<Type Name="StackOverflowException" FullName="System.StackOverflowException">
  <Metadata><Meta Name="ms.openlocfilehash" Value="660b017e3d3b3250c419111e5ff3713c5b94dcc9" /><Meta Name="ms.sourcegitcommit" Value="055a4a82a0b08bfbdc21bd1347fb71f7fe2c099e" /><Meta Name="ms.translationtype" Value="MT" /><Meta Name="ms.contentlocale" Value="de-DE" /><Meta Name="ms.lasthandoff" Value="08/15/2019" /><Meta Name="ms.locfileid" Value="69419307" /></Metadata><TypeSignature Language="C#" Value="public sealed class StackOverflowException : SystemException" />
  <TypeSignature Language="ILAsm" Value=".class public auto ansi serializable sealed beforefieldinit StackOverflowException extends System.SystemException" />
  <TypeSignature Language="DocId" Value="T:System.StackOverflowException" />
  <TypeSignature Language="VB.NET" Value="Public NotInheritable Class StackOverflowException&#xA;Inherits SystemException" />
  <TypeSignature Language="C++ CLI" Value="public ref class StackOverflowException sealed : SystemException" />
  <TypeSignature Language="F#" Value="type StackOverflowException = class&#xA;    inherit SystemException" />
  <AssemblyInfo>
    <AssemblyName>mscorlib</AssemblyName>
    <AssemblyVersion>1.0.5000.0</AssemblyVersion>
    <AssemblyVersion>2.0.0.0</AssemblyVersion>
    <AssemblyVersion>2.0.5.0</AssemblyVersion>
    <AssemblyVersion>4.0.0.0</AssemblyVersion>
  </AssemblyInfo>
  <AssemblyInfo>
    <AssemblyName>netstandard</AssemblyName>
    <AssemblyVersion>2.0.0.0</AssemblyVersion>
    <AssemblyVersion>2.1.0.0</AssemblyVersion>
  </AssemblyInfo>
  <AssemblyInfo>
    <AssemblyName>System.Runtime</AssemblyName>
    <AssemblyVersion>4.2.0.0</AssemblyVersion>
    <AssemblyVersion>4.2.1.0</AssemblyVersion>
  </AssemblyInfo>
  <Base>
    <BaseTypeName>System.SystemException</BaseTypeName>
  </Base>
  <Interfaces />
  <Attributes>
    <Attribute FrameworkAlternate="netframework-2.0;netframework-3.0;netframework-3.5;netframework-4.0;netframework-4.5;netframework-4.5.1;netframework-4.5.2;netframework-4.6;netframework-4.6.1;netframework-4.6.2;netframework-4.7;netframework-4.7.1;netframework-4.7.2;xamarinandroid-7.1;xamarinios-10.8;xamarinmac-3.0;netframework-4.8">
      <AttributeName>System.Runtime.InteropServices.ComVisible(true)</AttributeName>
    </Attribute>
    <Attribute FrameworkAlternate="netframework-1.1;netframework-2.0;netframework-3.0;netframework-3.5;netframework-4.0;netframework-4.5;netframework-4.5.1;netframework-4.5.2;netframework-4.6;netframework-4.6.1;netframework-4.6.2;netframework-4.7;netframework-4.7.1;netframework-4.7.2;netframework-4.8;xamarinandroid-7.1;xamarinios-10.8;xamarinmac-3.0">
      <AttributeName>System.Serializable</AttributeName>
    </Attribute>
  </Attributes>
  <Docs>
    <summary><span data-ttu-id="7dd52-101">Die Ausnahme, die bei einem Überlauf des Ausführungsstapels durch zu viele geschachtelte Methodenaufrufe ausgelöst wird.</span><span class="sxs-lookup"><span data-stu-id="7dd52-101">The exception that is thrown when the execution stack overflows because it contains too many nested method calls.</span></span> <span data-ttu-id="7dd52-102">Diese Klasse kann nicht vererbt werden.</span><span class="sxs-lookup"><span data-stu-id="7dd52-102">This class cannot be inherited.</span></span></summary>
    <remarks>
      <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="7dd52-103">`StackOverflowException`wird für Ausführungs Stapelüberlauf-Fehler ausgelöst, in der Regel bei einer sehr tiefen oder unbegrenzten Rekursion.</span><span class="sxs-lookup"><span data-stu-id="7dd52-103">`StackOverflowException` is thrown for execution stack overflow errors, typically in case of a very deep or unbounded recursion.</span></span> <span data-ttu-id="7dd52-104">Stellen Sie daher sicher, dass der Code nicht über eine Endlosschleife oder eine unendliche Rekursion verfügt.</span><span class="sxs-lookup"><span data-stu-id="7dd52-104">So make sure your code doesn't have an infinite loop or infinite recursion.</span></span>
  
 <span data-ttu-id="7dd52-105">`StackOverflowException`verwendet das HRESULT-COR_E_STACKOVERFLOW mit dem Wert 0x800703E9.</span><span class="sxs-lookup"><span data-stu-id="7dd52-105">`StackOverflowException` uses the HRESULT COR_E_STACKOVERFLOW, which has the value 0x800703E9.</span></span> <span data-ttu-id="7dd52-106">Die <xref:System.Reflection.Emit.OpCodes.Localloc> Il- `StackOverflowException`Anweisung (Intermediate Language) löst aus.</span><span class="sxs-lookup"><span data-stu-id="7dd52-106">The <xref:System.Reflection.Emit.OpCodes.Localloc> intermediate language (IL) instruction throws `StackOverflowException`.</span></span> <span data-ttu-id="7dd52-107">Eine Liste der anfänglichen Eigenschaftswerte für ein `StackOverflowException` -Objekt finden Sie <xref:System.StackOverflowException.%23ctor%2A> unter den-Konstruktoren.</span><span class="sxs-lookup"><span data-stu-id="7dd52-107">For a list of initial property values for a `StackOverflowException` object, see the <xref:System.StackOverflowException.%23ctor%2A> constructors.</span></span>  
  
 <span data-ttu-id="7dd52-108">Beginnend mit dem .NET Framework 2,0 können Sie `StackOverflowException` kein-Objekt mit einem `try` / `catch` -Block abfangen, und der entsprechende Prozess wird standardmäßig beendet.</span><span class="sxs-lookup"><span data-stu-id="7dd52-108">Starting with the .NET Framework 2.0, you can't catch a `StackOverflowException` object with a `try`/`catch` block, and the corresponding process is terminated by default.</span></span> <span data-ttu-id="7dd52-109">Folglich sollten Sie Ihren Code schreiben, um einen Stapelüberlauf zu erkennen und zu verhindern.</span><span class="sxs-lookup"><span data-stu-id="7dd52-109">Consequently, you should write your code to detect and prevent a stack overflow.</span></span> <span data-ttu-id="7dd52-110">Wenn Ihre APP z. b. von der Rekursion abhängig ist, verwenden Sie einen Counter-oder eine Status Bedingung, um die rekursive Schleife zu beenden.</span><span class="sxs-lookup"><span data-stu-id="7dd52-110">For example, if your app depends on recursion, use a counter or a state condition to terminate the recursive loop.</span></span>
<span data-ttu-id="7dd52-111">Eine Abbildung dieser Technik finden Sie im Abschnitt " [Beispiele](#examples) ".</span><span class="sxs-lookup"><span data-stu-id="7dd52-111">See the [Examples](#examples) section for an illustration of this technique.</span></span>
 
> [!NOTE]
>  <span data-ttu-id="7dd52-112">Das Anwenden <xref:System.Runtime.ExceptionServices.HandleProcessCorruptedStateExceptionsAttribute> des-Attributs auf eine Methode `StackOverflowException` , die eine auslöst, hat keine Auswirkungen.</span><span class="sxs-lookup"><span data-stu-id="7dd52-112">Applying the <xref:System.Runtime.ExceptionServices.HandleProcessCorruptedStateExceptionsAttribute> attribute to a method that throws a `StackOverflowException` has no effect.</span></span> <span data-ttu-id="7dd52-113">Die Ausnahme kann noch immer nicht aus dem Benutzercode behandelt werden.</span><span class="sxs-lookup"><span data-stu-id="7dd52-113">You still cannot handle the exception from user code.</span></span>

 <span data-ttu-id="7dd52-114">Wenn Ihre APP die Common Language Runtime (CLR) hostet, kann Sie angeben, dass die CLR die Anwendungsdomäne entladen soll, in der die Stapelüberlauf Ausnahme auftritt, und den entsprechenden Prozess fortsetzen lassen.</span><span class="sxs-lookup"><span data-stu-id="7dd52-114">If your app hosts the common language runtime (CLR), it can specify that the CLR should unload the application domain where the stack overflow exception occurs and let the corresponding process continue.</span></span> <span data-ttu-id="7dd52-115">Weitere Informationen finden Sie unter [ICLRPolicyManager-Schnittstelle](~/docs/framework/unmanaged-api/hosting/iclrpolicymanager-interface.md).</span><span class="sxs-lookup"><span data-stu-id="7dd52-115">For more information, see [ICLRPolicyManager Interface](~/docs/framework/unmanaged-api/hosting/iclrpolicymanager-interface.md).</span></span>

## Examples

 <span data-ttu-id="7dd52-116">Im folgenden Beispiel wird ein Leistungs Bezeichner verwendet, um sicherzustellen, dass die Anzahl `Execute` der rekursiven Aufrufe der-Methode nicht den von der MAX_RECURSIVE_CALLS-Konstante definierten Höchstwert überschreitet.</span><span class="sxs-lookup"><span data-stu-id="7dd52-116">The following example uses a counter to ensure that the number of recursive calls to the `Execute` method do not exceed a maximum defined by the MAX_RECURSIVE_CALLS constant.</span></span>
  
 [!code-csharp[System.StackOverflowException.Class#1](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.stackoverflowexception.class/cs/example1a.cs#1)]
 [!code-vb[System.StackOverflowException.Class#1](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.stackoverflowexception.class/vb/example1a.vb#1)]  

 ]]></format>
    </remarks>
    <altmember cref="T:System.Exception" />
    <related type="Article" href="~/docs/standard/exceptions/index.md"><span data-ttu-id="7dd52-117">Behandeln und Auslösen von Ausnahmen</span><span class="sxs-lookup"><span data-stu-id="7dd52-117">Handling and Throwing Exceptions</span></span></related>
  </Docs>
  <Members>
    <MemberGroup MemberName=".ctor">
      <AssemblyInfo>
        <AssemblyName>mscorlib</AssemblyName>
        <AssemblyVersion>2.0.5.0</AssemblyVersion>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Docs>
        <summary><span data-ttu-id="7dd52-118">Initialisiert eine neue Instanz der <see cref="T:System.StackOverflowException" />-Klasse.</span><span class="sxs-lookup"><span data-stu-id="7dd52-118">Initializes a new instance of the <see cref="T:System.StackOverflowException" /> class.</span></span></summary>
      </Docs>
    </MemberGroup>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public StackOverflowException ();" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor() cil managed" />
      <MemberSignature Language="DocId" Value="M:System.StackOverflowException.#ctor" />
      <MemberSignature Language="VB.NET" Value="Public Sub New ()" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; StackOverflowException();" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>mscorlib</AssemblyName>
        <AssemblyVersion>1.0.5000.0</AssemblyVersion>
        <AssemblyVersion>2.0.0.0</AssemblyVersion>
        <AssemblyVersion>2.0.5.0</AssemblyVersion>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>netstandard</AssemblyName>
        <AssemblyVersion>2.0.0.0</AssemblyVersion>
        <AssemblyVersion>2.1.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>System.Runtime</AssemblyName>
        <AssemblyVersion>4.2.0.0</AssemblyVersion>
        <AssemblyVersion>4.2.1.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters />
      <Docs>
        <summary><span data-ttu-id="7dd52-119">Initialisiert eine neue Instanz der <see cref="T:System.StackOverflowException" />-Klasse. Dabei wird die <see cref="P:System.Exception.Message" />-Eigenschaft der neuen Instanz auf eine vom System gelieferte Meldung festgelegt, die den Fehler beschreibt, z. B. "Dieser Vorgang verursachte einen Stapelüberlauf".</span><span class="sxs-lookup"><span data-stu-id="7dd52-119">Initializes a new instance of the <see cref="T:System.StackOverflowException" /> class, setting the <see cref="P:System.Exception.Message" /> property of the new instance to a system-supplied message that describes the error, such as "The requested operation caused a stack overflow."</span></span> <span data-ttu-id="7dd52-120">Diese Meldung berücksichtigt die aktuelle Systemkultur.</span><span class="sxs-lookup"><span data-stu-id="7dd52-120">This message takes into account the current system culture.</span></span></summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="7dd52-121">In der folgenden Tabelle werden die anfänglichen Eigenschaftenwerte für eine <xref:System.StackOverflowException>-Instanz aufgeführt.</span><span class="sxs-lookup"><span data-stu-id="7dd52-121">The following table shows the initial property values for an instance of <xref:System.StackOverflowException>.</span></span>  
  
|<span data-ttu-id="7dd52-122">Eigenschaft</span><span class="sxs-lookup"><span data-stu-id="7dd52-122">Property</span></span>|<span data-ttu-id="7dd52-123">Wert</span><span class="sxs-lookup"><span data-stu-id="7dd52-123">Value</span></span>|  
|--------------|-----------|  
|<xref:System.Exception.InnerException%2A>|<span data-ttu-id="7dd52-124">`null`.</span><span class="sxs-lookup"><span data-stu-id="7dd52-124">`null`.</span></span>|  
|<xref:System.Exception.Message%2A>|<span data-ttu-id="7dd52-125">Die lokalisierte Zeichenfolge mit der Fehlermeldung.</span><span class="sxs-lookup"><span data-stu-id="7dd52-125">The localized error message string.</span></span>|  
  
 ]]></format>
        </remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public StackOverflowException (string message);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string message) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.StackOverflowException.#ctor(System.String)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String)" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; StackOverflowException(System::String ^ message);" />
      <MemberSignature Language="F#" Value="new StackOverflowException : string -&gt; StackOverflowException" Usage="new System.StackOverflowException message" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>mscorlib</AssemblyName>
        <AssemblyVersion>1.0.5000.0</AssemblyVersion>
        <AssemblyVersion>2.0.0.0</AssemblyVersion>
        <AssemblyVersion>2.0.5.0</AssemblyVersion>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>netstandard</AssemblyName>
        <AssemblyVersion>2.0.0.0</AssemblyVersion>
        <AssemblyVersion>2.1.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>System.Runtime</AssemblyName>
        <AssemblyVersion>4.2.0.0</AssemblyVersion>
        <AssemblyVersion>4.2.1.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="message" Type="System.String" />
      </Parameters>
      <Docs>
        <param name="message"><span data-ttu-id="7dd52-126">Ein <see cref="T:System.String" />, der den Fehler beschreibt.</span><span class="sxs-lookup"><span data-stu-id="7dd52-126">A <see cref="T:System.String" /> that describes the error.</span></span> <span data-ttu-id="7dd52-127">Der Inhalt von message sollte für Personen verständlich sein.</span><span class="sxs-lookup"><span data-stu-id="7dd52-127">The content of message is intended to be understood by humans.</span></span> <span data-ttu-id="7dd52-128">Der Aufrufer dieses Konstruktors muss sicherstellen, dass diese Zeichenfolge für die aktuelle Systemkultur lokalisiert wurde.</span><span class="sxs-lookup"><span data-stu-id="7dd52-128">The caller of this constructor is required to ensure that this string has been localized for the current system culture.</span></span></param>
        <summary><span data-ttu-id="7dd52-129">Initialisiert eine neue Instanz der <see cref="T:System.StackOverflowException" />-Klasse mit einer angegebenen Fehlermeldung.</span><span class="sxs-lookup"><span data-stu-id="7dd52-129">Initializes a new instance of the <see cref="T:System.StackOverflowException" /> class with a specified error message.</span></span></summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="7dd52-130">In der folgenden Tabelle werden die anfänglichen Eigenschaftenwerte für eine <xref:System.StackOverflowException>-Instanz aufgeführt.</span><span class="sxs-lookup"><span data-stu-id="7dd52-130">The following table shows the initial property values for an instance of <xref:System.StackOverflowException>.</span></span>  
  
|<span data-ttu-id="7dd52-131">Eigenschaft</span><span class="sxs-lookup"><span data-stu-id="7dd52-131">Property</span></span>|<span data-ttu-id="7dd52-132">Wert</span><span class="sxs-lookup"><span data-stu-id="7dd52-132">Value</span></span>|  
|--------------|-----------|  
|<xref:System.Exception.InnerException%2A>|<span data-ttu-id="7dd52-133">Ein NULL-Verweis (`Nothing` in Visual Basic).</span><span class="sxs-lookup"><span data-stu-id="7dd52-133">A null reference (`Nothing` in Visual Basic).</span></span>|  
|<xref:System.Exception.Message%2A>|<span data-ttu-id="7dd52-134">Die Zeichenfolge der Fehlermeldung.</span><span class="sxs-lookup"><span data-stu-id="7dd52-134">The error message string.</span></span>|  
  
 ]]></format>
        </remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public StackOverflowException (string message, Exception innerException);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string message, class System.Exception innerException) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.StackOverflowException.#ctor(System.String,System.Exception)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String, innerException As Exception)" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; StackOverflowException(System::String ^ message, Exception ^ innerException);" />
      <MemberSignature Language="F#" Value="new StackOverflowException : string * Exception -&gt; StackOverflowException" Usage="new System.StackOverflowException (message, innerException)" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>mscorlib</AssemblyName>
        <AssemblyVersion>1.0.5000.0</AssemblyVersion>
        <AssemblyVersion>2.0.0.0</AssemblyVersion>
        <AssemblyVersion>2.0.5.0</AssemblyVersion>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>netstandard</AssemblyName>
        <AssemblyVersion>2.0.0.0</AssemblyVersion>
        <AssemblyVersion>2.1.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>System.Runtime</AssemblyName>
        <AssemblyVersion>4.2.0.0</AssemblyVersion>
        <AssemblyVersion>4.2.1.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="message" Type="System.String" />
        <Parameter Name="innerException" Type="System.Exception" />
      </Parameters>
      <Docs>
        <param name="message"><span data-ttu-id="7dd52-135">Die Fehlermeldung, in der die Ursache der Ausnahme erklärt wird.</span><span class="sxs-lookup"><span data-stu-id="7dd52-135">The error message that explains the reason for the exception.</span></span></param>
        <param name="innerException"><span data-ttu-id="7dd52-136">Die Ausnahme, die die Ursache der aktuellen Ausnahme ist.</span><span class="sxs-lookup"><span data-stu-id="7dd52-136">The exception that is the cause of the current exception.</span></span> <span data-ttu-id="7dd52-137">Wenn der <paramref name="innerException" />-Parameter kein NULL-Verweis ist (<see langword="Nothing" /> in Visual Basic), wird die aktuelle Ausnahme in einem <see langword="catch" />-Block ausgelöst, der die innere Ausnahme behandelt.</span><span class="sxs-lookup"><span data-stu-id="7dd52-137">If the <paramref name="innerException" /> parameter is not a null reference (<see langword="Nothing" /> in Visual Basic), the current exception is raised in a <see langword="catch" /> block that handles the inner exception.</span></span></param>
        <summary><span data-ttu-id="7dd52-138">Initialisiert eine neue Instanz der <see cref="T:System.StackOverflowException" />-Klasse mit einer angegebenen Fehlermeldung und einem Verweis auf die innere Ausnahme, die diese Ausnahme ausgelöst hat.</span><span class="sxs-lookup"><span data-stu-id="7dd52-138">Initializes a new instance of the <see cref="T:System.StackOverflowException" /> class with a specified error message and a reference to the inner exception that is the cause of this exception.</span></span></summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="7dd52-139">Eine Ausnahme, die als direktes Ergebnis einer vorhergehenden Ausnahme ausgelöst wird, kann in der <xref:System.Exception.InnerException%2A>-Eigenschaft über einen Verweis auf die vorhergehende Ausnahme verfügen.</span><span class="sxs-lookup"><span data-stu-id="7dd52-139">An exception that is thrown as a direct result of a previous exception can include a reference to the previous exception in the <xref:System.Exception.InnerException%2A> property.</span></span> <span data-ttu-id="7dd52-140">Die <xref:System.Exception.InnerException%2A>-Eigenschaft gibt denselben Wert zurück, der an den Konstruktor übergeben wurde, bzw. einen NULL-Verweis (`Nothing` in Visual Basic), wenn die <xref:System.Exception.InnerException%2A>-Eigenschaft nicht den Wert für die innere Ausnahme für den Konstruktor bereitstellt.</span><span class="sxs-lookup"><span data-stu-id="7dd52-140">The <xref:System.Exception.InnerException%2A> property returns the same value that is passed into the constructor, or a null reference (`Nothing` in Visual Basic) if the <xref:System.Exception.InnerException%2A> property does not supply the inner exception value to the constructor.</span></span>  
  
 <span data-ttu-id="7dd52-141">In der folgenden Tabelle werden die anfänglichen Eigenschaftenwerte für eine <xref:System.StackOverflowException>-Instanz aufgeführt.</span><span class="sxs-lookup"><span data-stu-id="7dd52-141">The following table shows the initial property values for an instance of <xref:System.StackOverflowException>.</span></span>  
  
|<span data-ttu-id="7dd52-142">Eigenschaft</span><span class="sxs-lookup"><span data-stu-id="7dd52-142">Property</span></span>|<span data-ttu-id="7dd52-143">Wert</span><span class="sxs-lookup"><span data-stu-id="7dd52-143">Value</span></span>|  
|--------------|-----------|  
|<xref:System.Exception.InnerException%2A>|<span data-ttu-id="7dd52-144">Der Verweis auf die interne Ausnahme.</span><span class="sxs-lookup"><span data-stu-id="7dd52-144">The inner exception reference.</span></span>|  
|<xref:System.Exception.Message%2A>|<span data-ttu-id="7dd52-145">Die Zeichenfolge der Fehlermeldung.</span><span class="sxs-lookup"><span data-stu-id="7dd52-145">The error message string.</span></span>|  
  
 ]]></format>
        </remarks>
        <altmember cref="T:System.Exception" />
        <related type="Article" href="~/docs/standard/exceptions/index.md"><span data-ttu-id="7dd52-146">Behandeln und Auslösen von Ausnahmen</span><span class="sxs-lookup"><span data-stu-id="7dd52-146">Handling and Throwing Exceptions</span></span></related>
      </Docs>
    </Member>
  </Members>
</Type>
